apps/console/src/features/feature-gate/models/feature-gate.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { FeatureGateAction } from "../actions/feature-gate";
+
+export interface FeatureGateShowInterface {
+    /**
+     * Permissions needed to render child elements
+     */
+    ifAllowed: string;
+}
+
+export interface FeatureGateContextInterface {
+    features: FeatureGateInterface;
+    dispatch: React.Dispatch<FeatureGateAction>;
+}
+
+
+export interface FeatureGateInterface {
+    console:  {
+        application: {
+            enabled: boolean,
+            signIn: {
+                adaptiveAuth : {
+                    enabled: boolean
+                },
+                enabled: boolean
+            }
+        },
+        enabled: boolean
+    }
```

Comment:
Though it bows to Lint practices, for readability purposes isn't it better to keep **enabled** as the first element in each? WDYT @thivi ?

Reply:
I think it is better to align with the lint rules. Since this is only an interface, I don't think readability should be an issue.

Reply:
Ack.